external-crates/move/crates/move-package/src/resolution/dependency_graph.rs

```diff
@@ -1214,12 +1214,8 @@ impl DependencyGraph {
     /// This operation fails, writing nothing, if the graph contains a cycle, and can fail with an
     /// undefined output if it cannot be represented in a TOML file.
     pub fn write_to_lock(&self, install_dir: PathBuf) -> Result<LockFile> {
-        let lock = LockFile::new(
-            install_dir,
-            self.manifest_digest.clone(),
-            self.deps_digest.clone(),
-        )?;
-        let mut writer = BufWriter::new(&*lock);
+        use fmt::Write;
```

Comment:
nit: I would personally just put the trait `use` statements at the top-level (also applies to `Seek` below)

Reply:
For sure! In this case, there is already a `io::Write` at the top that conflicts. I'm not sure there's an elegant way to overcome that?